client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -286,6 +287,10 @@ public class ClientOptions
     @Option(names = "--disable-compression", description = "Disable compression of query results")
     public boolean disableCompression;
 
+    @PropertyMapping(ENCODING_ID)
```

Comment:
I do not like the `EncodingId`. Looks like something which is per-query, not type of encoding.

I think just `encoding` or `coded` would be better name.

Also would be nice if help returned list of available values.

Reply:
this is per query, you can have a different encoding on each query

Reply:
it's consistent with the verb used across the code. In the future versions this will have a default value so users won't need to set it up.

Reply:
<img width="1068" alt="Screenshot 2024-09-09 at 10 20 40" src="https://github.com/user-attachments/assets/ffe978c6-333e-455a-af1e-44381ce463e6">

Good enough?

Reply:
> this is per query, you can have a different encoding on each query

Yeah - by per query I mean it looks like some encoding instance id. Not just type you can pick from the enum. 

Docs changes looks fine of course. But I would rename the encoding-id to be just encoding here and in codebase as a separate commit after all those changes.

Reply:
Ok, I've noted this as a follow up